src/components/TranslationPanel.js

```diff
@@ -11,11 +11,17 @@ const i18n = new(require('../translations/i18n'));
 const db = require(`${__dirname}/../util/data-provider`).targetDb();
 import Statistic  from '../components/Statistic';
 import { FormattedMessage } from 'react-intl';
+import { CommunicationSpeakerPhone } from 'material-ui/svg-icons';
 
+var el;
```

Comment:
I can't see where these are used. Please clean-up unused variables before merging.

Reply:
Yes i will recheck and clear all the unused before PR